mlflow/server/handlers.py

```diff
@@ -1917,6 +1921,117 @@ def _delete_artifact_mlflow_artifacts(artifact_path):
     return response
 
 
+@catch_mlflow_exception
+@_disable_unless_serve_artifacts
+def _create_multipart_upload_artifact(artifact_path):
+    """
+    A request handler for `POST /mlflow-artifacts/mpu/create` to create a multipart upload
+    to `artifact_path` (a relative path from the root artifact directory).
+    """
+    validate_path_is_safe(artifact_path)
+
+    request_message = _get_request_message(
+        CreateMultipartUpload(),
+        schema={
+            "path": [_assert_required, _assert_string],
+            "num_parts": [_assert_intlike],
+        },
+    )
+    path = request_message.path
+    num_parts = request_message.num_parts
+
+    artifact_repo = _get_artifact_repo_mlflow_artifacts()
+    if not isinstance(artifact_repo, MultipartUploadMixin):
+        raise MlflowException(
+            "Multipart upload is not supported for artifact repository "
+            f"{artifact_repo.__class__.__name__}",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
```

Comment:
i tried refactoring this out, but intellisense won't be able to detect the object type

Reply:
> intellisense won't be able to detect the object type

Can you elaborate?

Reply:
don't worry about it, I extracted it out to a method